src/bitcoind_rpc_api.rs

```diff
@@ -0,0 +1,126 @@
+use bitcoin::{Address, BlockHash, Transaction, Txid};
+use bitcoincore_rpc_json::{
+    FinalizePsbtResult, GetAddressInfoResult, GetBlockResult, GetBlockchainInfoResult,
+    GetDescriptorInfoResult, GetTransactionResult, GetWalletInfoResult, ListUnspentResultEntry,
+    LoadWalletResult, WalletCreateFundedPsbtResult,
+};
+use serde::{Deserialize, Serialize};
+use std::collections::HashMap;
+
+#[jsonrpc_client::api]
+#[async_trait::async_trait]
```

Comment:
You shouldn't be needing this, the macro will put the attribute on the trait that is generated!

Reply:
If you do need this, file a bug report please :D